addons/mail/static/src/core/common/composer.xml

```diff
@@ -237,4 +237,26 @@
             <t t-esc="option.emoji.shortcodes.join(' ')"/>
         </em>
     </t>
+
+    <t t-name="mail.Suggestion.Partner">
+        <a t-att-href="href" t-att-data-oe-id="partnerId" data-oe-model="res.partner" target="_blank" contenteditable="false" draggable="false"
+           class="o_mail_redirect">@<t t-esc="displayName"/>
+        </a>
+    </t>
+
+    <t t-name="mail.Suggestion.Special">
+        <a t-att-href="href" target="_blank" contenteditable="false" draggable="false" class="o-discuss-mention">@<t t-esc="label"/>
+        </a>
+    </t>
+
+    <t t-name="mail.Suggestion.Role">
+        <a t-att-href="href" t-att-data-oe-id="roleId" data-oe-model="res.role" target="_blank" contenteditable="false" draggable="false"
+           class="o-discuss-mention">@<t t-esc="displayName"/>
+        </a>
```

Comment:
If we use multi-line for the link, we should have the inside text node be on its own (indented) line instead.
It would make the template easier to read, there is no point to just have the closing `a` on its own line.